src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/caller/CalledCopyRatioSegment.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.caller;
+
+import org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation.CopyRatioSegment;
+import org.broadinstitute.hellbender.utils.Utils;
+
+public class CalledCopyRatioSegment extends CopyRatioSegment {
+    public enum Call {
+        AMPLIFICATION_CALL("+"),
```

Comment:
Given that the instances of this enum will always be reference with the enum name itself i.e. as `Call.AMPLIFICATION_CALL`, I think you can safely rename them to `AMPLIFICATION`, `DELETION`, and `NEUTRAL` with no loss of clarity.

Reply:
Good "call".